Missing module `yargs`

When trying to run the `masterchat-cli`, the following error is thrown:

```
masterchat --filter 'superchat' 7MxfQ6IcQHM
internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module 'yargs'
Require stack:
- /usr/local/lib/node_modules/masterchat-cli/lib/cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/masterchat-cli/lib/cli.js:8:33)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/usr/local/lib/node_modules/masterchat-cli/lib/cli.js' ]
}
```

Node version: tried both 14 and 17
OS: macOS 12.0.1 21A559 x86_64

--- a/package.json
+++ b/package.json
@@ -23,7 +23,8 @@
   "dependencies": {
     "epicfail": "^3.0.0",
     "masterchat": "^0.11.0",
-    "vm2": "^3.9.3"
+    "vm2": "^3.9.3",
+    "yargs": "^17.2.1"
   },
   "devDependencies": {
     "@types/jest": "^27.0.1",

